autopush/router/apns2.py

```diff
@@ -117,16 +117,14 @@ def send(self, router_token, payload, apns_id,
                 raise RouterException(
                     "APNS Transmit Error {}:{}".format(response.status,
                                                        reason),
-                    status_code=500,
+                    status_code=502,
                     response_body="APNS could not process "
-                                  "your message {}".format(reason))
+                                  "your message {}".format(reason)
+                )
         except HTTP20Error as ex:
             connection.close()
-            raise RouterException(
-                "APNS Processing error: {}".format(repr(ex)),
-                status_code=503,
-                response_body="APNS returned an error processing request",
-            )
+            self.log.error("APNS Processing error: {}".format(repr(ex)))
```

Comment:
If there's a http2 error, I'd want the report as close to the actual source of the error as possible. An unexpected and unhandled error kills the connection and potentially loses the request.
Until we have a better understanding of the sorts of errors we should see when dealing with http2 and apns, I'm fine with getting sentry errors that describe them when they happen.